doc/guides/dts/howtos.rst

```diff
@@ -659,20 +659,22 @@ node with path ``/soc/i2c@12340000`` in a C/C++ file:
 
 And if you're trying to **set** that property in a devicetree overlay:
 
-.. code-block:: DTS
+.. code-block:: devicetree
 
-   /*
-    * foo.overlay: DTS names with special characters, etc.
-    */
+   \{
```

Comment:
Why is this here? I thought it would not be necessary to artificially put things under the root node anymore

Reply:
This is the only case that is not handled. Note that current lexer doesn't seem to process it correctly either:

![image](https://user-images.githubusercontent.com/25011557/117046961-9ae9e080-ad11-11eb-9a67-cd11183d07be.png)

But for some reason it doesn't generate any warning.
https://docs.zephyrproject.org/latest/guides/dts/howtos.html#make-sure-you-re-using-the-right-names

Reply:
How about just `code-block:: none` then?

Reply:
with current patch:

![image](https://user-images.githubusercontent.com/25011557/117047514-42671300-ad12-11eb-9b50-6da98c9aebd0.png)

Reply:
Putting this inside of the root node is just actively wrong for what this example is trying to be about